documentation/sphinx/source/release-notes/release-notes-620.rst

```diff
@@ -4,6 +4,10 @@
 Release Notes
 #############
 
+6.2.34
+======
+* Add the ``trace_partial_file_suffix`` network option. This option will give unfinished trace files a special suffix to indicate they're not complete yet. When the trace file is complete, it is renamed to remove the suffix. `(PR #5349) <https://github.com/apple/foundationdb/pull/5349>`_
+
```

Comment:
I'm not sure what we should do with this on future branches. Typically we would add it on master, but also I don't think we're planning to actually cut a 6.2.34.

Reply:
Good point. Given that we don't plan to cut 6.2.34 I think this is probably ok, but I'm happy to add this release note to master and cherry-pick down to release-7.0 and release-6.3 if you think that's desirable